terraform/modules/query-service-infra/vpc.tf

```diff
@@ -0,0 +1,141 @@
+//
```

Comment:
Can you share more about why you chose to create a vpc per env? If you do create vpcs for these, it would be best to use shared-infra's modules so that we are not maintaining our own VPC configuration.

https://github.com/chanzuckerberg/shared-infra/tree/master/terraform/modules/aws-env

Reply:
reverted to using the default vpc